manifests/plugin/pulp/params.pp

```diff
@@ -3,4 +3,5 @@
 class foreman_proxy::plugin::pulp::params {
   $pulpcore_api_url     = "https://${facts['networking']['fqdn']}"
   $pulpcore_content_url = "${pulpcore_api_url}/pulp/content"
+  $rhsm_url             = "https://${facts['networking']['fqdn']}/rhsm"
```

Comment:
The current default here would be:

```suggestion
  $rhsm_url             = "https://${facts['networking']['fqdn']}:8443/rhsm"
```

And we are eventually aiming to arrive at this being 443 by default. That leaves us with that question of, should this reflect the current default? The future default? or be left blank so that what configures this plugin has to set this value?

Reply:
Ahh yea, that is fair to default to the "default" setup. Would you mind opening the corresponding puppet-foreman_proxy_content change just so we have a working example of this being implemented?

Reply:
puppet-FPC is here: https://github.com/theforeman/puppet-foreman_proxy_content/pull/384